ProcessLib/ComponentTransport/ComponentTransportFEM.h

```diff
@@ -85,14 +97,20 @@ class LocalAssemblerData : public ComponentTransportLocalAssemblerInterface
                        std::size_t const local_matrix_size,
                        bool is_axially_symmetric,
                        unsigned const integration_order,
-                       ComponentTransportProcessData const& process_data)
+                       ComponentTransportProcessData const& process_data,
+                       std::vector<std::vector<std::reference_wrapper<ProcessVariable>>> const&
```

Comment:
I think it is possible to pass
```std::vector<std::reference_wrapper<ProcessVariable>> const& process_variables```
instead of `std::vector<std::vector<std::reference_wrapper<ProcessVariable>>>`.
Then you could write later on (for instance in line 113):
```_process_variables.size()```.

Reply:
_process_variables is derived from the base class of Process. And if using std::vector<std::reference_wrapper<ProcessVariable>> const& process_variables instead, we may encounter some problems when using staggered scheme.

Reply:
Dropped the two-dimensional vector.